src/Illuminate/Database/Eloquent/Relations/Pivot.php

```diff
@@ -83,6 +93,28 @@ protected function setKeysForSaveQuery(Builder $query)
     }
 
     /**
+     * Eager load any defined relations.
+     *
+     * @return null
+     */
+    protected function eagerLoadRelations()
+    {
+        foreach ($this->with as $relation) {
+            $this->relations[$relation] = $this->$relation()->getResults();
+        }
+    }
+
+    /**
+     * Retrieve columns required by custom pivot model.
+     *
+     * @return array
+     */
+    public static function getPivotColumns()
+    {
+        return (new ReflectionClass(static::class))->getDefaultProperties()['include'];
```

Comment:
Yes. Tested the syntax on 5.5.34 and 5.5.5 here: http://sandbox.onlinephpfunctions.com/code/6951dbfc1d81443f7ff0fbf3b7333c247fa09a82